compat/src/index.js

```diff
@@ -117,8 +117,58 @@ const flushSync = (callback, arg) => callback(arg);
  */
 const StrictMode = Fragment;
 
+const startTransition = callback => callback();
+const useTransition = () => {
+	const [isPending, setIsPending] = useState(false);
+
+	const start = cb => {
+		setIsPending(true);
+		cb();
+		setIsPending(false);
```

Comment:
I don’t understand what this is meant to do. Won’t all updates be batched here anyway?